compiler/Main.hs

```diff
@@ -87,7 +87,7 @@ cppCodegen options@Cpp {..} = do
     templates = concat $ map snd $ filter fst codegen_templates
     codegen_templates = [ (core_enabled, core_files)
                         , (comm_enabled, [comm_h export_attribute, comm_cpp])
-                        , (grpc_enabled, [grpc_h export_attribute])
+                        , (grpc_enabled, [grpc_h export_attribute, grpc_cpp])
```

Comment:
We need to update the cmake to include the generated *grpc.cpp files, in this line: https://github.com/Microsoft/bond/blob/master/cmake/Bond.cmake#L60

Reply:
We also need to compile them. Later on in the same file, in `add_bond_executable`, something similar to what's done for Comm is needed now.

Reply:
Got it.